h/static/scripts/vendor/jquery.scrollintoview.js

```diff
@@ -109,29 +109,37 @@
 
 				var animOptions = {};
 
+				var yDir = "none";
```

Comment:
> I suspect that forking to our own repo is probably the cleanest option at this stage.

OK, I moved our form from my github repo to here: https://github.com/hypothesis/jquery-scrollintoview.
The preamble of the JS source file (in /vendor) now mentions this URL.

Is it OK now?